### PR TITLE
Adding warntoggle to the examples folder

### PR DESCRIPTION
**Pull request by robdejonge**
Following up on https://github.com/jpmens/mqttwarn/issues/404, adding a custom function to be used when configuring streams in `mqttwarn`, `warntoggle` makes it easy to toggle notifications on or off through a simple web interface.

### CODE REVIEW

File: examples/warntoggle/mqttwarn/customfunctions.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+    try:
+        with open(filename) as infile:    
+            toggles = json.load(infile)
+            infile.close()
+
+        if topic in toggles: 
+            # file found, topic found
+            topicblock = toggles[topic]
+	    srv.logging.debug('togglestate() was called from the ' + section + ' section and found ' + topic + ' in ' + filename)
```
Comment (jpmens) [inline examples/warntoggle/mqttwarn/customfunctions.py:17]: I think there's an indentation problem here. Is that a tab?

File: examples/warntoggle/README.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- This custom function checks a JSON file for the topic, and based on ``TRUE`` or ``FALSE`` setting will return a message back to `mqttwarn` to either allow the notification to happen or not.
+- If an MQTT topic was not found in aforementioned JSON file, it will be added and set with a configurable default value.
+- For the user, an accompanying web script allows the toggle to be changed.
+
+Installation
+============
+
+- Copy the ``togglestate()`` function from ``mqttwarn/customfunctions.py`` to your own custom functions file, or copy the entire file and refer to it in `mqttwarn.ini` with a  ``functions = 'customfunctions.py'`` directive.
+- Copy the content of the `www` folder to a web server on the same host as `mqttwarn`, ensure Python is enabled for the server.
+- Create a symbolic link from ``/opt/mqttwarn/warntoggle.json`` -> ``/var/www/html/warntoggle.json`, obviously adjusted for your local situation. Alternatively, configure the filename inside the custom function where it now says ``filename = "warntoggle.js"`` to contain an absolute path.
```
Comment (jpmens) [inline examples/warntoggle/README.rst:21]: please add something like "and ensure ``warntoggle.json`` is writeable by the HTTP server.

Comment (robdejonge) [reply examples/warntoggle/README.rst:21]: Done.

File: examples/warntoggle/mqttwarn/customfunctions.py (hunk 1 above)
Comment (robdejonge) [reply examples/warntoggle/mqttwarn/customfunctions.py]: Yep. Corrected.

**Comment by robdejonge** [comment]
Not very experienced on GitHub, but I think the additional changes are now also part of this pull request.

**Comment by jpmens** [comment]
Merged. Thank you for your contribution!

**Pull request merged**